internal/sysfs/dirfs.go

```diff
@@ -93,10 +78,11 @@ func (d *dirFS) Utimes(name string, atimeNsec, mtimeNsec int64) error {
 }
 
 func (d *dirFS) join(name string) string {
-	if name == "." {
-		return d.cleanedHostDir
+	switch name {
```

Comment:
windows is more sensitive to this than others, but basically there's no need to add a trailing delimiter when requesting the root path.